packages/yourloops/app/main-lobby.tsx

```diff
@@ -84,40 +76,43 @@ export function MainLobby(): JSX.Element {
   const isCurrentRoutePublic = PUBLIC_ROUTES.includes(currentRoute)
   const isCurrentRouteAlwaysAccessible = ALWAYS_ACCESSIBLE_ROUTES.includes(currentRoute)
   const theme = getTheme()
-  const { palette } = useTheme()
-  const classes = routeStyle({
-    color: EXTERNAL_THEME_ROUTES.includes(currentRoute) ? palette.background.default : palette.background.paper
-  })
+  const classes = routeStyle()
```

Comment:
So we no longer need to compute the color based on the route?

Reply:
Not anymore, the background is now the same faor the whole application, it's computed in the theme.tsx file